phmi/prefix.py

```diff
@@ -0,0 +1,13 @@
+def strip_prefix(s):
```

Comment:
this feels like something that could go wrong. Do you want to do something like

`int(prefix.strip)`

(TBH, your more familiar with the data, just  a suggestion)

Reply:
Which part do you think could go wrong here? It's hard to tell because you've commented on the definition, not a part of the function 😉 

That being said, this function takes the string `2. Foo` and returns `Foo` but I notice it will fail if the string doesn't contain `.` so I'll add a shortcut check for that.